lbz/authz/collector.py

```diff
@@ -0,0 +1,16 @@
+from lbz.misc import Singleton
+
+
+class AuthCollector(metaclass=Singleton):
+    # TODO: Add uni tests
+    def __init__(self) -> None:
+        self.possible_authz: list = []
+
+    def add_authz(self, permission: str) -> None:
+        self.possible_authz.append(permission)
```

Comment:
The dump method bases on the resource but here you are adding all permissions into one list, I think that is not fully correct 🤔